src/lib/components/atoms/Modal.svelte

```diff
@@ -0,0 +1,64 @@
+<script>
+    import { click_outside } from "$lib/utils"
+    import { fade } from 'svelte/transition';
+    let modalOpen = false;
+    export let icon = '';
+</script>
+
+<div role="dialog" aria-modal="true" class="modal">
```

Comment:
Can you move `aria-modal` and `role="dialog"` to the modal container?
Also, `class="modal"` is confusing because it tricks you into thinking it is the actual modal...